chroot/seccomp.go

```diff
@@ -21,7 +21,7 @@ func setSeccomp(spec *specs.Spec) error {
 	mapAction := func(specAction specs.LinuxSeccompAction, errnoRet *uint) libseccomp.ScmpAction {
 		switch specAction {
 		case specs.ActKill:
-			return libseccomp.ActKill
```

Comment:
I've applied it here because the CI test "Smoke Test" was failing, perhaps because updating `containers/storage` bought an update to `seccomp/libseccomp-golang`. Perhaps updating `containers/storage` should be a separate PR?